.bazelrc_shared

```diff
@@ -22,14 +22,3 @@ test --test_output=all
 
 build:rules --disk_cache=.bazel_cache
```

Comment:
Will this option still take effect, now that we've removed the `common --config=rules` line?